upgrade/v15xto160/upgrade.go

```diff
@@ -160,6 +164,55 @@ func upgradeVolumeAttachments(namespace string, lhClient *lhclientset.Clientset,
 	return nil
 }
 
+func upgradeDefaultPriorityClass(namespace string, lhClient *lhclientset.Clientset, resourceMaps map[string]interface{}) (err error) {
```

Comment:
We don't need to do this instead of just relying on the chart values (or manifest)

Reply:
Removed the upgrade path for the default Priority Class.